youtube_dl/extractor/nieuwsblad.py

```diff
@@ -0,0 +1,93 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import re
+
+from .common import InfoExtractor
+from ..utils import (
+    smuggle_url
+)
+
+
+class NieuwsbladIE(InfoExtractor):
+    """ Extractor for www.nieuwsblad.be """
+    _VALID_URL = r'https?://(?:www\.)?nieuwsblad\.be/.+?/dmf([0-9]+?)_(?P<id>[0-9]+)'
+    _TESTS = [
+        # Source: VMMA
+        {
+            'url': 'http://www.nieuwsblad.be/cnt/dmf20151224_02036890',
+            'md5': '3dcf2c3a140d8e54dd8376d4c4a609f4',
+            'info_dict': {
+                'id': '02036890',
+                'ext': 'mp4',
+                'title': 'Krijgt zieke Pauline (3) het mooiste kerstcadeau?',
+                'thumbnail': 're:http.*jpg$',
+            }
+        },
+        # Source: VRT
+        {
+            'url': 'http://www.nieuwsblad.be/cnt/dmf20151124_01986463',
+            'md5': '8e46cb7ddfddeb64735fa39f105002c2',
+            'info_dict': {
+                'id': '01986463',
+                'ext': 'mp4',
+                'title': 'Angst voor terreur: fotograaf toont hoe hij de werkelijkheid kan manipuleren',
+                'thumbnail': 're:http.*jpg$',
+            }
+        },
+        # Source: Mediahuis (using kaltura)
+        {
+            'url': 'http://www.nieuwsblad.be/cnt/dmf20151225_02037264',
+            'md5': 'd4decdc7f105c26767b928c54c7d5184',
+            'info_dict': {
+                'id': '1_z4jndqki',
+                'ext': 'mov',
+                'title': 'autobrand Peer',
+                'thumbnail': 're:^https?://.*/thumbnail/.*',
+                'timestamp': int,
+                'upload_date': '20151225',
+                'uploader_id': 'dcc-video-manager-hbvl@mediahuis.be'
+            }
+        },
+    ]
+
+    def _real_extract(self, url):
+        """ Extract the video info from the given 'nieuwsblad' URL """
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+
+        iframe_m = re.search(r'<script[^>]+src="(.+?kaltura.com.*?)"', webpage)
+        if iframe_m:
+            return self._extract_kaltura(url, webpage)
+
+        thumbnail = self._og_search_thumbnail(webpage)
+        title = self._og_search_title(webpage)
+
+        iframe_m = re.search(r'<iframe[^>]+src="(.+?vrt.be.*?)"', webpage)
+        if iframe_m:
+            webpage = self._download_webpage(iframe_m.group(1), "vrt-iframe")
+            video_url = self._search_regex(r'sources.pdl = "(.*?)";', webpage, 'vrt-video')
+
+        iframe_m = re.search(r'<iframe[^>]+src="(.+?vmma.be.*?)"', webpage)
+        if iframe_m:
+            webpage = self._download_webpage(iframe_m.group(1), "vmma-iframe")
+            video_url = self._search_regex(r'<source src="(.*?)"', webpage, 'vmma-video')
```

Comment:
Parsing of vmma.be can be deferred to medialaan.py. This code works: https://github.com/rg3/youtube-dl/commit/67763af8595882485766bbb01c99897c14dd2c9e
@yan12125 The only thing I've never understood is the importance of video-ids for youtube-dl. Could you clarify, I can't find any documentation on it? In this example (www.nieuwsblad.be/cnt/dmf20151224_02036890) the referring page has an 'article id' of 02036890. The article embeds a vmma video using a 'vID' (https://syndication.medialaan.be/syndication/?vID=50534bb11fe5827eb25cad5c3445c5ae&sID=HNB&autoplay=false). But parsing that page, the json contains a video id "id":122781. So which one to use?
Please check https://github.com/rg3/youtube-dl/commit/67763af8595882485766bbb01c99897c14dd2c9e#R151, I don't know if this change is acceptable or not.

When it comes to mediahuis.py, the vmma link can then be deferred to mediahuis.py. Code (I added limburger.nl as well): https://github.com/rg3/youtube-dl/commit/9b34d3ede9e66c15f51468579db18a755e59809f
Yen, could you check https://github.com/rg3/youtube-dl/commit/9b34d3ede9e66c15f51468579db18a755e59809f#diff-fa9395c261329be01699b4ddff476e9cR168? Which is the preferred way of deferring the url, line 167 or line 168?